utils/format.go

```diff
@@ -525,6 +525,18 @@ func FormatWithdawalCredentials(hash []byte, addCopyButton bool) template.HTML {
 	return text
 }
 
+func FormatAddressToWithdawalCredentials(address []byte, addCopyButton bool) template.HTML {
+	combined := fmt.Sprintf("0x010000000000000000000000%v", fmt.Sprintf("%#x", address)[2:])
+
+	var text template.HTML = template.HTML(fmt.Sprintf("<a href=\"/address/%#x\"><span class=\"text-monospace text-success\">0x01</span><span class=\"text-monospace\">00…%x</span></a>", address, address[len(address)-2:]))
+
+	if addCopyButton {
+		text += template.HTML(fmt.Sprintf("<i class=\"fa fa-copy text-muted p-1\" role=\"button\" data-toggle=\"tooltip\" title=\"Copy to clipboard\" data-clipboard-text=\"%s\"></i>", combined))
+	}
+
+	return text
+}
+
```

Comment:
This can be replaced by something like this:

```
func FormatAddressToWithdawalCredentials(address []byte, addCopyButton bool) template.HTML {
	credentials, err := hex.DecodeString("010000000000000000000000")
	if err != nil {
		return "INVALID CREDENTIALS"
	}
	credentials = append(credentials, address...)

	return FormatWithdawalCredentials(credentials, addCopyButton)
}
```

That way, credential formatting logic like the coloring (`text-success` and `text-warning`) is not duplicated.

Please change and test.

Reply:
Thanks, works great!